tests/cpp/test_matmul_aten_evaluation.cpp

```diff
@@ -334,4 +340,84 @@ TEST_F(MatmulATenEvaluationTest, MatmulBiasAlpha) {
 
   EXPECT_TRUE(at::allclose(out[0], out_ref));
 }
+
+// fd.ops.linear(a, b) where a = [M,K], b = [N,K]
+TEST_F(MatmulATenEvaluationTest, Linear) {
```

Comment:
I think it might be useful to have another test where we call at::linear with only 2 arguments (no bias).